BraveRewardsUI/Tipping/TippingView.swift

```diff
@@ -11,6 +12,39 @@ extension TippingViewController {
     
     var gesturalDismissExecuted: (() -> Void)?
     
+    func setInfo(name: String, tipAmount: Double, recurringDate: String?) {
```

Comment:
Lets rename this to something that conveys its for the confirmation view, and lets also make it `private` since its not used anywhere outside of this scope.

Maybe like `updateConfirmationInfo` or something